src/main/java/net/dv8tion/jda/api/interactions/commands/CommandInteraction.java

```diff
@@ -200,4 +200,52 @@ default OptionMapping getOption(@Nonnull String name)
         List<OptionMapping> options = getOptionsByName(name);
         return options.isEmpty() ? null : options.get(0);
     }
+
+    /**
+     * Gets the slash command String for this slash command.
+     * <br>This is similar to the String you see when clicking the interaction name in the client.
+     *
+     * <p>Example return for an echo command: {@code /say echo phrase: Say this}
+     *
+     * @return The command String for this slash command
+     */
+    @Nonnull
+    default String getCommandString()
+    {
+        //Get text like the text that appears when you hover over the interaction in discord
+        StringBuilder builder = new StringBuilder();
+        builder.append("/").append(getName());
+        if (getSubcommandGroup() != null)
+            builder.append(" ").append(getSubcommandGroup());
+        if (getSubcommandName() != null)
+            builder.append(" ").append(getSubcommandName());
+        for (OptionMapping o : getOptions())
+        {
+            builder.append(" ").append(o.getName()).append(":");
```

Comment:
I'm realizing now that we could do this
```suggestion
            builder.append(" ").append(o.getName()).append(": ");
```
and remove the preceding `.append(" ")` from all of the lines below this one.

Reply:
change made in code, waiting for push